portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-details.js

```diff
@@ -0,0 +1,43 @@
+const { format, parseISO } = require('date-fns');
+
+/**
+ * @typedef {Object} ReportAndUserDetails
+ * @property {string} uploadedByFullName - The index of the object with format '{firstname} {surname}'
+ * @property {string} formattedDateAndTime - The date uploaded formatted as 'h:mmaaa'
```

Comment:
`formattedDateAndTime` is too generic in this context as there are a few different dates relating to a report and period. Maybe `formattedUploadDateAndTime`?

Reply:
I went with `formattedDateAndTimeUploaded`